src/main/java/training/endava/skeleton/Main.java

```diff
@@ -40,16 +47,90 @@ public static void main(String[] args) {
         personList.forEach(System.out::println);
         personRepository.deleteAll();
         personList.forEach(System.out::println);
+
+
+        List<Company> companies = createDummyDataCompany();
+        System.out.println("----------------Ex1------------------");
+        companies.forEach(System.out::println);
```

Comment:
The task was to list the names of all companies. You can either map the list of companies to their names or override toString so that it only prints the names. The first solution is preferred so that you don't change the behavior of a class method just for a certain task, because you might need to print something else later and you can't change that again because this task will be impacted and so on.